RPT_Application/src/main/java/com/harman/web_pages/UserReportCRUDView.java

```diff
@@ -0,0 +1,171 @@
+package com.harman.web_pages;
+
+import org.springframework.beans.factory.annotation.Autowired;
+
+import com.harman.report_database.Report;
+import com.harman.report_database.ReportService;
+import com.harman.report_database.Task;
+import com.harman.report_database.TaskService;
+import com.harman.user_database.Employee;
+import com.harman.user_database.EmployeeService;
+import com.vaadin.flow.component.button.Button;
+import com.vaadin.flow.component.datepicker.DatePicker;
+import com.vaadin.flow.component.dialog.Dialog;
+import com.vaadin.flow.component.formlayout.FormLayout;
+import com.vaadin.flow.component.grid.Grid;
+import com.vaadin.flow.component.grid.ItemDoubleClickEvent;
+import com.vaadin.flow.component.html.H3;
+import com.vaadin.flow.component.html.Label;
+import com.vaadin.flow.component.icon.Icon;
+import com.vaadin.flow.component.icon.VaadinIcon;
+import com.vaadin.flow.component.listbox.ListBox;
+import com.vaadin.flow.component.orderedlayout.HorizontalLayout;
+import com.vaadin.flow.component.textfield.TextArea;
+import com.vaadin.flow.router.Route;
+import com.vaadin.flow.spring.annotation.UIScope;
+
+@Route("my_reports")
+@UIScope
+public class UserReportCRUDView extends DefaultReportCRUDView {
+
+	private static final long serialVersionUID = 5606092593469344695L;
+
+	@Autowired
+	ReportService reportService;
+
+	@Autowired
+	EmployeeService employeeService;
+
+	@Autowired
+	TaskService taskService;
+
+	private H3 wwbH3;
+
+	public UserReportCRUDView(ReportService reportService, EmployeeService employeeService) {
+		super(reportService, employeeService);
+
+		super.getGrid().addItemDoubleClickListener(e -> taskDialog(super.getGrid(), e, super.getEmployee()));
+
+		Button addButton = new Button("Add Report");
+		addButton.setWidth("250px");
+		addButton.setHeight("75px");
+
+		addButton.addClickListener(e -> createSaveDialog(super.getGrid(), super.getEmployee()));
+		add(addButton);
+	}
+
+	private void taskDialog(Grid<Report> grid, ItemDoubleClickEvent<Report> e, Employee employee) {
+		Dialog dialog = new Dialog();
+		HorizontalLayout layout = new HorizontalLayout();
+		FormLayout buttomLayout = new FormLayout();
+
+		layout.setJustifyContentMode(JustifyContentMode.BETWEEN);
+		layout.add(new Label("Month: " + e.getItem().getDate()));
+		Icon exitIcon = new Icon(VaadinIcon.CLOSE);
+		exitIcon.getStyle().set("cursor", "pointer");
+		exitIcon.setColor("red");
+		exitIcon.addClickListener((__) -> dialog.close());
+		layout.add(exitIcon);
+
+		Button addTaskButton = new Button("Add Task");
+		addTaskButton.setHeight("75px");
+		addTaskButton.setWidth("250px");
+
+		addTaskButton.addClickListener(event -> {
+			createTask(TaskCRUDView.getGrid(), e.getItem(), employee);
+		});
+
+		wwbH3 = new H3();
+		wwbH3.setText("WWB: " + e.getItem().getWwb());
```

Comment:
Why h3 not any of enum editing fields?

Reply:
i think that H3 gives this nice feling of inputing something to the application :3
also .getWwb() returns String in xxx.xx% format